<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmaddCoffee</class>
 <widget class="QWidget" name="FrmaddCoffee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>216</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>170</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Сорт кофе - добавление строки</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>-51</x>
     <y>49</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Обжарка   </string>
   </property>
  </widget>
  <widget class="QComboBox" name="CBRoast">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>58</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>5</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Название</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>57</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Обжарка   </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>109</y>
     <width>33</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Объем</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CBForm">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>33</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>83</y>
     <width>26</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Цена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnOK">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>31</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Форма (в зернах и т.д.)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editPrice">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>81</y>
     <width>250</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editValue">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>107</y>
     <width>250</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editName">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>7</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editDescr">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>137</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>137</y>
     <width>48</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Описание</string>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>CBRoast</zorder>
  <zorder>BtnCancel</zorder>
  <zorder>label_9</zorder>
  <zorder>label_5</zorder>
  <zorder>label_8</zorder>
  <zorder>label_7</zorder>
  <zorder>BtnOK</zorder>
  <zorder>label_2</zorder>
  <zorder>CBForm</zorder>
  <zorder>editPrice</zorder>
  <zorder>editValue</zorder>
  <zorder>editName</zorder>
  <zorder>editDescr</zorder>
  <zorder>label_10</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
